connector_search_engine/models/se_index.py

```diff
@@ -65,6 +76,19 @@ def recompute_all_binding(self, force_export=False):
                 bindings._jobify_recompute_json(force_export=force_export)
         return True
 
+    @api.onchange('lang_id', 'model_id', 'backend_id.name')
+    def _compute_name(self):
+        ctx = self.env.context
+        for rec in self:
+            backend = rec.backend_id
+            if not backend and ctx.get('model') and ctx.get('backend_id'):
+                backend = rec.env[ctx['model']].browse(ctx['backend_id'])
```

Comment:
@bealdav You could also initialize backend_id

Reply:
Done